test/tools/d_do_test.d

```diff
@@ -590,7 +597,6 @@ int tryMain(string[] args)
             default:      envData.ccompiler = "c++"; break;
         }
     }
-    bool msc = envData.ccompiler.toLower.endsWith("cl.exe");
```

Comment:
I would just change this line to `envData.hasMicrosoftCompiler = envData.ccompiler.toLower.endsWith("cl.exe");`